bigchaindb/utils.py

```diff
@@ -185,3 +186,23 @@ def load_node_key(path):
         priv_key = priv_validator['priv_key']['value']
         hex_private_key = key_from_base64(priv_key)
         return key_pair_from_ed25519_key(hex_private_key)
+
+
+def check_tendermint_version(running_tm_ver):
+    """
+    Check Tendermint compatability with BigchainDB server
+
+    :param running_tm_ver: Version number of the connected Tendermint instance
+    :type running_tm_ver: str
+    :return: True/False depending on the compatability with BigchainDB server
+    :rtype: bool
+    """
+    tm_ver = running_tm_ver.split('-')
```

Comment:
Maybe put a comment with an example value of `running_tm_ver` so it's clear why you're doing this split.